plsql/PlSqlParser.g4

```diff
@@ -701,6 +701,18 @@ inline_constraint
       constraint_state?
     ;
 
+inline_ref_constraint
+    : SCOPE IS tableview_name
+      | WITH ROWID
```

Comment:
Vertical bar `|` for the first-level alternatives should strictly follow after the colon `:`. Here and further.

```antlr
inline_ref_constraint
    : SCOPE IS tableview_name
    | WITH ROWID
```

Reply:
that feedback is particularly appreciated, wasn't sure how to handle that situation